### PR TITLE
Vehicle faker

### PR DESCRIPTION
**Pull request by snoozins**

### CODE REVIEW

**Comment by snoozins** [comment]
Not sure how to cover that stray line in the CheckSum generator
Thoughts welcome.

File: tests/FakerDotNet.Tests/Fakers/RandomFakerTests.cs [hunk 1]
```diff
@@ -1,65 +1,65 @@
-using System.Linq;
```
Comment (mrstebo) [inline tests/FakerDotNet.Tests/Fakers/RandomFakerTests.cs]: Not sure why it says that this file has changed :sweat_smile:

File: src/FakerDotNet/Algorithms/VinCheckSum.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+namespace FakerDotNet.Algorithms
```
Comment (snoozins) [inline src/FakerDotNet/Algorithms/VinCheckSum.cs]: I've dropped this into the namespace, but we can move it, it doesn't feel quite right, as it's not an algorithm.

File: tests/FakerDotNet.Tests/Fakers/RandomFakerTests.cs (hunk 1 above)
Comment (snoozins) [reply tests/FakerDotNet.Tests/Fakers/RandomFakerTests.cs]: I can't seem to revert the non change :/
Might have been a line ending change.

File: src/FakerDotNet/Data/VehicleData.cs [hunk 3]
```diff
@@ -0,0 +1,492 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace FakerDotNet.Data
+{
+    internal static class VehicleData
+    {
+        internal static readonly IEnumerable<string> Manufactures = new[]
```
[the remark this answers is not in the document]
Comment (snoozins) [reply src/FakerDotNet/Data/VehicleData.cs]: Done

File: tests/FakerDotNet.Tests/Fakers/VehicleFakerTests.cs [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+
+        private IFakerContainer _fakerContainer;
+        private IVehicleFaker _vehicleFaker;
+        private IRandomWrapper _randomWrapper;
+
+        [Test]
+        public void Make_returns_a_make()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(
+                    A<IEnumerable<string>>.That.IsSameSequenceAs(VehicleData.Makes)))
```
[the remark this answers is not in the document]
Comment (snoozins) [reply tests/FakerDotNet.Tests/Fakers/VehicleFakerTests.cs]: Much cleaner!
Done as well

File: src/FakerDotNet/Algorithms/VinCheckSum.cs (hunk 2 above)
Comment (mrstebo) [reply src/FakerDotNet/Algorithms/VinCheckSum.cs]: Maybe there could be a Checksum namespace? I think there may be a few areas where we need to generate checksums, so having them all grouped together in the same namespace would be beneficial 🤔

Also, should the class be called VinChecksum?

**Pull request merged**